samples/snapshot/README.md

```diff
@@ -0,0 +1,4 @@
+# Pub/Sub Lite Spark Connector Word Count Sample
+
+This directory contains samples for Pub/Sub Lite Spark Connector.
+
```

Comment:
Ideally we need a full how-to guide like this: https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/pubsub/streaming-analytics/README.md

But we can address that in a separate PR.

Reply:
Updated the readme.